earth_enterprise/src/scons/getversion.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2.7
+#!/usr/bin/env python
```

Comment:
This is failing for me in el6 because the build environment now is supposed to be setup for python2.7 and not the default python for el6.  those following this rule will get failures.